cmd/kops/rollingupdatecluster.go

```diff
@@ -83,6 +87,8 @@ To perform rolling update, you need to update the cloud resources first with "ko
 	cmd.Flags().DurationVar(&options.NodeInterval, "node-interval", options.NodeInterval, "Time to wait between restarting nodes")
 	cmd.Flags().DurationVar(&options.BastionInterval, "bastion-interval", options.BastionInterval, "Time to wait between restarting bastions")
 
+	cmd.Flags().Int64Var(&options.Headroom, "headroom", options.Headroom, "Additional nodes to provision before starting rolling update (default 0)")
```

Comment:
Should we calculate this for the user?  Also we need to add a node per AZ ... PVC storage ... darn ebs volumes